vault/resource_database_secret_backend_connection.go

```diff
@@ -430,6 +463,16 @@ func mysqlConnectionStringResource() *schema.Resource {
 	return r
 }
 
+func mssqlConnectionStringResource() *schema.Resource {
+	r := connectionStringResource(&connectionStringConfig{})
+	r.Schema["contain_db"] = &schema.Schema{
```

Comment:
Should this be `contained_db` instead since that is what Vault expects
https://github.com/hashicorp/vault/blob/main/plugins/database/mssql/mssql.go#L102

Reply:
Maybe I'm misunderstanding, but I see `contain_db` as part of the schema, though it is not being set and it's not being read from the user either. What are we doing with this field?

Reply:
Great catch!

Reply:
Fixed in https://github.com/hashicorp/terraform-provider-vault/pull/1259/commits/e4f13218baf4429206bd6b4340d6d1dc3f7ea1a2